src/django/api/templates/mail/claim_facility_denial_body.txt

```diff
@@ -16,5 +16,5 @@ The claim information is inaccurate. If you believe this denial to be inaccurate
 
```

Comment:
On line 15 we include the info@ email in the body text. I would suggest changing it to data@ to harmonize with the from address.

Reply:
https://github.com/open-apparel-registry/open-apparel-registry/pull/2373/commits/1c2f378882f981eb0eef9b80b82cc613d2a8041b